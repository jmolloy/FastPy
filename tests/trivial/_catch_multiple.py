try:
    print x
except (NotImplementedError, RuntimeError) as e:
    print e
