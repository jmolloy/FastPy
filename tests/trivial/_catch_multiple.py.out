Key does not exist: "x"
